app/controllers/images_controller.rb

```diff
@@ -23,7 +23,7 @@ def redirect_to_image(type)
       if @parent.image.attached?
         redirect_to url_for(@parent.send(type))
       else
-        raise ActionController::RoutingError.new("Image Not Found")
+        raise ActionController::RoutingError.new("Image Not Attached")
```

Comment:
Tests?

Reply:
Beyond this: spec/suport/imageable_spec.rb:27 what should we test for? The text?